installation/bclinux-setup.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```

Comment:
would be nice to have a bclinux-installation-guide.md in https://github.com/clearcontainers/runtime/tree/master/docs
to include it in the wiki

Reply:
Hi,@devimc,I have written a document of bclinux-installation which I 'm going to submit.